.gitignore

```diff
@@ -45,3 +45,5 @@ docs/build/
 
 # Python
 .venv/
+
+config/paraglider_config.yaml
```

Comment:
Where did this `config/` folder come from?

Reply:
My bad. This is a local folder I added for glided. Should have removed it before PR'ing. Sending a fix